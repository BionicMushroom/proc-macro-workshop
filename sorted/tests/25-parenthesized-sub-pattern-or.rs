#[allow(clippy::unnested_or_patterns, unused_parens)]
#[sorted::check]
fn main() {
    let x = 5;

    #[sorted]
    match x {
        5 => println!("5"),
        (8 | 9) | (6 | 7) => println!("6, 7, 8 or 9"),
        _ => println!("other"),
    }
}
