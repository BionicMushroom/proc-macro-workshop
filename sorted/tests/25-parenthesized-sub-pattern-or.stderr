error: `(6 | 7)` should sort before `(8 | 9)`
 --> tests/25-parenthesized-sub-pattern-or.rs:9:19
  |
9 |         (8 | 9) | (6 | 7) => println!("6, 7, 8 or 9"),
  |                   ^^^^^^^
